Jenkinsfile

```diff
@@ -0,0 +1,84 @@
+stage('Preparation') {
+    node('builder') {
+        deleteDir()
+        git branch: 'jfbuild', url: 'https://github.com/codilime/contrail-windows-tools/'
```

Comment:
Unfortunatelly, I've hardcoded the branch name here. The correct way would be to inherit the branch from where the `Jenkinsfile` is checked out.
Anyway, I think we should merge as is and fix it in PR #110.

Reply:
Is Jenkinsfile configured to be taken from gerrit/gh branch right now?